scripts/npm-stats.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import { JSDOM } from 'jsdom';
+
+async function getStatsForPackage(packageName) {
+  const { data } = await axios.get(
+    `https://www.npmjs.com/package/${packageName}?activeTab=versions`
+  );
+
+  const dom = new JSDOM(data);
+  const ul = dom.window.document.querySelector(
+    '.a0dff0b1 ~ .c495d29d ~ .a0dff0b1 ~ .c495d29d'
```

Comment:
```suggestion
    '#tabpanel-versions > * > ul:last-child'
```

I find this a bit more speaking (and hopefully less likely to break)